### PR TITLE
Done ur Cleanup

### PR DESCRIPTION
**Pull request by HelloHowIsItGoing**

### CODE REVIEW

File: src/CefBrowser/Runtime/WebBrowserClient.cs [hunk 1]
```diff
@@ -419,15 +416,15 @@ private void HandleCefProcessLog(object sender, DataReceivedEventArgs e)
...
+        #region Browser Controlls
 
         /// <summary>
         ///     Sends a keyboard event to the CEF process
         /// </summary>
         /// <param name="keysDown"></param>
         /// <param name="keysUp"></param>
         /// <param name="chars"></param>
-        internal void SendKeyboardEvent(int[] keysDown, int[] keysUp, string chars)
+        public void SendKeyboardControlls(int[] keysDown, int[] keysUp, string chars)
```
Comment (Voltstro) [inline src/CefBrowser/Runtime/WebBrowserClient.cs:427]: Controls, not Controlls :)
(Same with above)

**Review by Voltstro — CHANGES_REQUESTED** [review]
Epic spelling

File: src/CefBrowser/Runtime/WebBrowserClient.cs (hunk 1 above)
Comment (HelloHowIsItGoing) [reply src/CefBrowser/Runtime/WebBrowserClient.cs:427]: Bruh

Comment (Voltstro) [reply src/CefBrowser/Runtime/WebBrowserClient.cs:427]: Well if you want what ur want from me, I except the best lol.

Comment (HelloHowIsItGoing) [reply src/CefBrowser/Runtime/WebBrowserClient.cs]: yea yea yea, there you go.

**Review by Voltstro — COMMENTED** [review]
Epic

**Comment by HelloHowIsItGoing** [comment]
There you go, now give me my shit.

**Pull request merged**